### ISSUE TITLE
keybase-gui doesn't start

### ISSUE THREAD

**Issue by MichaelAkvo** [NixOS/nixpkgs#177872]
### Describe the bug

Simply installing `keybase-gui` doesn't allow it to start up. There's no visual indication as to what the error is.

### Steps To Reproduce
1. Install `keybase-gui`
2. Try to start keybase GUI from the desktop
3. ...

### Expected behavior

The keybase GUI to show up.

### Screenshots

`-`

### Additional context

First run

```sh
$ keybase-gui 
Keybase service doesn't seem to be running.
You might need to run: keybase service
Set NIX_SKIP_KEYBASE_CHECKS=1 if you want to skip this check.
```

Setting `services.keybase.enable = true;` doesn't help as it doesn't show up in the units of the system nor of the user `systemctl [--user] list-unites | grep keybase`.

I had to actually run `keybase service`. Unfortunately, that didn't help either as the next error was 

```
$ keybase-gui 
Could not find kbfsfuse client in keybase status.
You might need to run: kbfsfuse
Set NIX_SKIP_KEYBASE_CHECKS=1 if you want to skip this check.
```

Setting `services.kbfs.enable = true;` doesn't help either - probably because the `keybase` service won't start. So I had to run `kbfsfuse` myself and finally it worked.

Is nix not able to install and start all the required services by simply installing `keybase-gui`?

### Notify maintainers

@avaq @rvolosatovs @puffnfresh @np @Br1ght0ne @shofius

### Metadata
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 5.10.111, NixOS, 21.11 (Porcupine)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.3.16`
 - channels(root): `"nixos-21.11.337100.7b38b03d76a"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

**Comment by peterhoeg** [comment]
Isn't keybase (as a project) dead after zoom buy-out?

**Comment by Avaq** [comment]
@peterhoeg they're still maintaining it: https://github.com/NixOS/nixpkgs/pull/177248

**Comment by fabianhjr** [comment]
There haven't been new features / new experiments but it still being maintained upstream.

**Comment by fabianhjr** [comment]
BTW @MichaelAkvo, to confirm some information:

Are you running on x11 or wayland?

On wayland electron apps have to be run without sandboxing to avoid a known issue on NixOS: `keybase-gui --no-sandbox` would start if it is the case.

From a terminal you could also check the output of `echo $XDG_SESSION_TYPE` (in my case `wayland`)

**Comment by MichaelAkvo** [comment]
@fabianhjr 

```
$ echo $XDG_SESSION_TYPE
x11
```

I believe that regardless of windowing system, the application will still be unhappy about the missing services.

After restarting, I now see the `keybase` service 

```
$ systemctl --user status keybase.service 
● keybase.service - Keybase service
     Loaded: loaded (/etc/systemd/user/keybase.service; enabled; vendor preset: enabled)
     Active: active (running) since Mon 2022-06-20 09:29:15 CEST; 3min 1s ago
```

I'm guessing the services don't have a step to reload the `systemd` daemon after installation.

Regardless, it would be great to have these services installed and run with the installation of `keybase-gui`, without having to manually find out which services a package needs in order to run. Is this a limitation of `nix` or nixOS itself? Is this just missing from the package?

**Comment by JoNilsson** [comment]
it is indeed a bit of leg-work to get keybase services running on NixOS.  
Just ensure that `keybase service` & `kbfsfuse` are running.  You can then run the `keybase-gui` and login like normal.  
But then it just works seamlessly with no issue.  I just execute a script that get's it all going with my window manager sub-system.

**Comment by Avaq** [comment]
Yeah, this hasn't been an issue for ages. I've been running Keybase-Gui on and off without problems all throughout the last 3 years.
